Makefile

```diff
@@ -317,7 +321,7 @@ MAP_SIM_TOP = $(CORE_SIM_TOP)
 
 # There's always a mapped Verilog file since there's macros in it and I'm only
 # going to bother generating macros as Verilog.
-OBJ_MAP_RTL_V = $(OBJ_SOC_RTL_V) $(OBJ_MAP_DIR)/plsi-generated/$(MAP_TOP).macros_for_synthesis.v
+OBJ_MAP_RTL_V = $(OBJ_SOC_RTL_V) $(OBJ_MAP_DIR)/plsi-generated/$(MAP_TOP).macros_for_synthesis.v $(TECHNOLOGY_VERILOG_SYNTHESIS_FILES)
```

Comment:
Added example for how to use it. Will add the list of all variables to my TODO list.